### DISCUSSION TITLE
Know More About EddieHub

### DISCUSSION THREAD

**Discussion by verma-kunal** [EddieHubCommunity/support#1548] [Show and tell]
Hello ! 
**Welcome to EddieHub!** We are glad to have you in our Community !!😃

Want to know more about EddieHub and how you can collaborate & contribute, be sure to read this article written by our very own **Hubber**

Link: https://dev.to/kaiwalyakoparkar/accelerate-your-learning-and-grow-your-network-with-eddiehub-1icn

**Comment by ghost** [comment]
Amazing Post! Thanks, @kaiwalyakoparkar and @verma-kunal!

**Comment by Arun-kc** [comment]
Great Post!! @kaiwalyakoparkar and @verma-kunal

**Reply by verma-kunal** [reply]
All thanks to **@kaiwalyakoparkar** for writing that !!
I just created this discussion for new folks !!😃

**Reply by Arun-kc** [reply]
Everything counts!! @verma-kunal 👍

**Comment by rohan-kulkarni-25** [comment]
Great Post @verma-kunal  !! Nice Article @kaiwalyakoparkar

**Reply by kaiwalyakoparkar** [reply]
Thank you very much for your kind words :heart:

**Comment by shikharcodes** [comment]
Great idea @verma-kunal! Very useful article for every new member joining the community, nicely explained @kaiwalyakoparkar 👍🏻

**Reply by kaiwalyakoparkar** [reply]
Thank you very much for you kind words :smile:

**Comment by Dhriti-Roy** [comment]
Great post @verma-kunal !!
